main.tf

```diff
@@ -0,0 +1,171 @@
+locals {
+  name_prefix = "${var.name_prefix}apps-gpdb-"
+}
+
+resource "aws_subnet" "dq_database" {
+  vpc_id                  = "${var.appsvpc_id}"
+  cidr_block              = "${var.dq_database_cidr_block}"
+  map_public_ip_on_launch = false
+  availability_zone       = "${var.az}"
+
+  tags {
+    Name = "${local.name_prefix}subnet"
+  }
+}
+
+module "gpdb_master1" {
+  source          = "github.com/UKHomeOffice/connectivity-tester-tf"
+  subnet_id       = "${aws_subnet.dq_database.id}"
+  user_data       = "CHECK_self=127.0.0.1:8080 CHECK_google=google.com:80 CHECK_googletls=google.com:443 LISTEN_tcp=0.0.0.0:5432 LISTEN_tcp2=0.0.0.0:9000 LISTEN_tcp3=0.0.0.0:22 LISTEN_tcp4=0.0.0.0:28090 CHECK_segment1:${var.gpdb_segment1_ip}:1024 CHECK_segment1:${var.gpdb_segment1_ip}:10000 CHECK_segment1:${var.gpdb_segment1_ip}:20000 CHECK_segment2:${var.gpdb_segment2_ip}:1024 CHECK_segment2:${var.gpdb_segment2_ip}:10000 CHECK_segment2:${var.gpdb_segment2_ip}:20000 CHECK_segment3:${var.gpdb_segment3_ip}:1024 CHECK_segment3:${var.gpdb_segment3_ip}:10000 CHECK_segment3:${var.gpdb_segment3_ip}:20000 CHECK_segment4:${var.gpdb_segment4_ip}:1024 CHECK_segment4:${var.gpdb_segment4_ip}:10000 CHECK_segment4:${var.gpdb_segment4_ip}:20000 CHECK_segment5:${var.gpdb_segment5_ip}:1024 CHECK_segment5:${var.gpdb_segment5_ip}:10000 CHECK_segment5:${var.gpdb_segment5_ip}:20000"
```

Comment:
do you need to test for internet egress to google here? also no need to test itself

Reply:
Changes commited.